Libraries/acdriver/Sources/CompileActionImageSet.cpp

```diff
@@ -259,10 +259,7 @@ CompileAsset(
     std::string name = parent->name().string();
 
     /* scale defaults to 0 / all */
-    double scale = 0;
-    if (image.scale()) {
-        scale = image.scale()->value();
-    }
+    double scale = image.scale()->value();
```

Comment:
Oh, my bad, this won't work. If `image.scale()` doesn't exist, it'll be `ext::nullopt` and crash on dereference. I thought that was of type `ext::optional<double>`, not `ext::optional<xcassets::Slot::Scale>`. Sorry!